packages/commonwealth/client/index.html

```diff
@@ -3,27 +3,21 @@
   <head>
     <meta charset="utf-8" />
     <title>Common</title>
-    <base href="/" />
-    <link rel="stylesheet" href="https://use.typekit.net/axy6pzh.css" />
```

Comment:
why did we remove `<link rel="stylesheet" href="https://use.typekit.net/axy6pzh.css" />`?

Reply:
Good catch, moving it back in. But we should really refactor this in order to make it a local file, instead of depending on someone else's server.